File: wrappers/java/src/main/java/org/hyperledger/indy/sdk/WalletClosedException.java
package org.hyperledger.indy.sdk;

public class WalletClosedException extends IndyException
{
	private final static String message = "The wallet is closed and cannot be used.";

	WalletClosedException(int sdkErrorCode) 
    {
    	super(message, sdkErrorCode);
    }
}

File: wrappers/dotnet/indy-sdk-dotnet/WalletApi/WalletClosedException.cs
namespace Hyperledger.Indy.WalletApi
{
    /// <summary>
    /// Exception thrown when an attempt is made to use a closed wallet.
    /// </summary>
    public class WalletClosedException : IndyException
    {
        const string message = "The wallet is closed and cannot be used.";

        /// <summary>
        /// Initializes a new WalletClosedException.
        /// </summary>
        internal WalletClosedException() : base(message, (int)ErrorCode.WalletInvalidHandle)
        {

        }
    }

}
